03-transactions.md

```diff
@@ -686,21 +789,38 @@ The *expected weight* of a commitment transaction is calculated as follows:
 			....htlc_output's...
 		- lock_time: 4 bytes
 
+	 commitment_transaction (option_anchor_outputs): 223 + 43 * num-htlc-outputs bytes
+		- version: 4 bytes
+		- witness_header <---- part of the witness data
+		- count_tx_in: 1 byte
+		- tx_in: 41 bytes
+			funding_input
+		- count_tx_out: 3 byte
+		- tx_out: 172 + 43 * num-htlc-outputs bytes
+			output_paying_to_remote,
+			output_paying_to_local,
+			output_anchor,
+			output_anchor,
+			....htlc_output's...
+		- lock_time: 4 bytes
+
 Multiplying non-witness data by 4 results in a weight of:
 
-	// 500 + 172 * num-htlc-outputs weight
+	// 500 + 172 * num-htlc-outputs weight (no option_anchor_outputs)
+	// 900 + 172 * num-htlc-outputs weight (option_anchor_outputs)
```

Comment:
223 * 4 = 892, not 900 :confused: 
Is this calculation up-to-date with the latest changes?

Reply:
Yes. I suspect the difference is that you didn't increase the bytes for `count_tx_out` from 1 to 3.

Reply:
Right, so it's the line that says `223 + 43 * num-htlc-outputs bytes` that is incorrect, it should say `225 + 43 * num-htlc-outputs bytes`?

Reply:
It is indeed incorrect. Pushed fix.

Reply:
Ok I get it, what confused me is that fact that for old commitments we use only 1 byte for `count_tx_out` whereas we should use 3 bytes in the worst case (where there are many HTLC outputs). Now everything makes sense :)

Reply:
Yes, I added that change not so long ago. But good that we are aligned again and thanks for pointing out the error in the pr.